tools/text_processing/add_line_to_file/add_line_to_file.xml

```diff
@@ -0,0 +1,51 @@
+<tool id="add-line-to-file" name="Add line to file" version="0.1.0">
+    <description>writes a line of text at the begining or end of a text file.</description>
+    <requirements>
+        <requirement type="package" version="8.25">coreutils</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+    <![CDATA[
+        #if $options == 'header':
+            echo '$text_input' >> '$outfile' && cat '$infile' >> '$outfile'
+        #else
+            cat '$infile' >> '$outfile' && echo '$text_input' >> '$outfile'
+        #end if
+    ]]>
+    </command>
+    <inputs>
+        <param name="text_input" type="text" format="txt" label="text to add"/>
+        <param name="infile" type="data" format="txt" label="input file"/>
+        <param name="kjh" type="data" format="txt" label="input file"/>
```

Comment:
haha, this is nice

Reply:
lol sorry I forgot to remove the bullshit testing